### DISCUSSION TITLE
Indo do escopo da Main() para o escopo do namespace .NET5 vs .NET6

### DISCUSSION THREAD

**Discussion by bitPIMP** [balta-io/forum#24] [Tira Dúvidas] [answered]
Curso: Fundamentos C#
Aula: Mão na massa: Criando uma calculadora
Sessão: Utilizando funções

Dúvida: Mestre @andrebaltieri , como eu saio do método main e vou para escopo do namespace da minha aplicação no .NET6 se no .NET6 essas delimitações de escopo estão ocultas? Como eu crio algo análogo ao void Soma() dentro do escopo do namespace que contém o main? 

![Tela da Aula  NET5](https://user-images.githubusercontent.com/94226995/192118740-4b7120e2-5ff2-4762-ac46-d03ca64ff09b.jpg)
Imagem 1: Tela da Aula

![Minha Tela  NET6](https://user-images.githubusercontent.com/94226995/192118743-b970feac-6551-45a1-8099-6edb4b47048e.jpg)
Imagem 2: Minha Tela

**Comment by andrebaltieri** [comment, marked as answer]
@bitPIMP bom dia! Você ainda pode ter um namespace com o nome da sua aplicação e um método Main... o código do .NET 5 é compatível no 6.

Porém, se não quiser, basta chamar um método inicial qualquer no `Program.cs` que vai funcionar.
